cosmic-applet-audio/src/pulse.rs

```diff
@@ -219,6 +219,7 @@ impl PulseHandle {
                 loop {
                     // This is where the we match messages from the GUI to pass to the pulse server
                     let mut msgs = Vec::new();
+                    msgs.push(to_pulse_recv.recv().await);
                     while let Ok(msg) = to_pulse_recv.try_recv() {
                         msgs.push(msg);
```

Comment:
Maybe this could search for the position of an existing element before pushing, instead of collecting all messages and then deduplication.

Reply:
not sure to understand this part of the code (cc @BrunoWallner), but this patch seems to fix the regression

Reply:
Oh, oops I made a mistake, sorry. The patch still does what I originally intended, so you are right.